[2025-05-06T16:56:02.214+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_airflow.print_hello scheduled__2025-03-29T00:00:00+00:00 [queued]>
[2025-05-06T16:56:02.222+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_airflow.print_hello scheduled__2025-03-29T00:00:00+00:00 [queued]>
[2025-05-06T16:56:02.223+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-06T16:56:02.414+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): print_hello> on 2025-03-29 00:00:00+00:00
[2025-05-06T16:56:02.426+0000] {standard_task_runner.py:60} INFO - Started process 1101 to run task
[2025-05-06T16:56:02.431+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'hello_***', 'print_hello', 'scheduled__2025-03-29T00:00:00+00:00', '--job-id', '572', '--raw', '--subdir', 'DAGS_FOLDER/example.py', '--cfg-path', '/tmp/tmpiowfn78r']
[2025-05-06T16:56:02.434+0000] {standard_task_runner.py:88} INFO - Job 572: Subtask print_hello
[2025-05-06T16:56:02.454+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-06T16:56:02.505+0000] {task_command.py:423} INFO - Running <TaskInstance: hello_airflow.print_hello scheduled__2025-03-29T00:00:00+00:00 [running]> on host afa31441e313
[2025-05-06T16:56:02.620+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_***' AIRFLOW_CTX_TASK_ID='print_hello' AIRFLOW_CTX_EXECUTION_DATE='2025-03-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-29T00:00:00+00:00'
[2025-05-06T16:56:02.623+0000] {logging_mixin.py:188} INFO - Hello, Airflow! DAG đang chạy thành công.
[2025-05-06T16:56:02.624+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-06T16:56:02.639+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=hello_***, task_id=print_hello, execution_date=20250329T000000, start_date=20250506T165602, end_date=20250506T165602
[2025-05-06T16:56:02.684+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-06T16:56:02.774+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
